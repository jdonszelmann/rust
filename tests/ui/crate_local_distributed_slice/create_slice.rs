#![feature(crate_local_distributed_slice)]
// @build-pass

#[distributed_slice(crate)]
const MEOWS: [&str; _];

fn main() {}
