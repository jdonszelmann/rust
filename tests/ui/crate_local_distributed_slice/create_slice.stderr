error: free constant item without body
  --> $DIR/create_slice.rs:5:1
   |
LL | const MEOWS: [&str; _];
   | ^^^^^^^^^^^^^^^^^^^^^^-
   |                       |
   |                       help: provide a definition for the constant: `= <expr>;`

error[E0121]: the placeholder `_` is not allowed within types on item signatures for constants
  --> $DIR/create_slice.rs:5:21
   |
LL | const MEOWS: [&str; _];
   |                     ^ not allowed in type signatures

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0121`.
